src/components/ChecklistPanelContent.js

```diff
@@ -57,6 +65,16 @@ const ChecklistPanelContent = ( {
 					{ requiredLabel }
 				</p>
 			) }
+			{ ( ! shouldBlockPublish && ! isComplete ) && (
+				<ToggleControl
+					label={ __( 'This post is ready to publish', 'altis-publication-checklist' ) }
+					help={ __( 'You have incomplete tasks remaining, please confirm that you are ready to publish this post.', 'altis-publication-checklist' ) }
```

Comment:
```suggestion
					help={ __( 'You have incomplete tasks remaining. Confirm that you wish to ignore these tasks and publish anyway.', 'altis-publication-checklist' ) }
```

This would actually benefit from using the same logic in ConfirmPublishButtonLabel to set the verb correctly here, as it's not always publishing.

Reply:
Updated with a component based off the confirm button label one to provide contextual help text.